spec/fluent.asdl

```diff
@@ -23,10 +23,9 @@
 
 module Fluent
 {
-    res = Resource(entry* body, comment? comment, span? span)
+    res = Resource(entry* body, span? span)
```

Comment:
What is the expected outcome of "file-level" comment being placed in the middle of the file? Say:

```properties
key = Message

# Message level comment
key2 = Message 2

### Suddenly, a wild file-level message appears
key3 = Message 3
```

Reply:
It appears in the AST a such: a `Comment` entry between `key2` and `key3`.